scripts/dump-connectors.js

```diff
@@ -20,10 +20,15 @@ const listFile = `${resDir}/connectors.json`;
 
 async function main(args) {
 	const latestTag = args[2];
-	let exitCode = 0;
 
+	if (latestTag === undefined) {
```

Comment:
It could be simplified to `if (!latestTag)`.

Reply:
thnx :)